c2rust-transpile/src/rust_ast/set_span.rs

```diff
@@ -186,7 +186,7 @@ set_span_impl!(enum ImplItem, s via
 
 impl SetSpan for Block {
     fn set_span(&mut self, s: Span) {
-        if self.stmts.is_empty() == false {
+        if !self.stmts.is_empty() {
```

Comment:
just my opinion, but i think clippy `bool_comparison` lint is wrong here. `!self.stmts.is_empty()` is double negation and using `== false` is more obvious. 
(https://github.com/rust-lang/rust-clippy/issues/4717)

Reply:
i think its ok to follow default suggestion from clippy in `c2rust` project. I write like that `if self.stmts.is_empty() == false {` because habit and i usualy set `bool_comparison` to allow in all my rust project 😉

Reply:
`== false` feels wrong to me, although `match` `bool` I like even though `clippy` doesn't.  Not really a big deal, though, so I just follow `clippy`.